notebook/static/notebook/less/cell.less

```diff
@@ -1,59 +1,66 @@
 @_cell_padding_minus_border: @cell_padding - @cell_border_width;
 
-._marked_style(@n) {
-    border-left-width: @n;
-    padding-left: @cell_padding - @n;
+._selected_style(@c1, @c2, @sep:0, @border_width:@cell_border_width) {
+    border-left-width: @border_width;
+    padding-left: @cell_padding - @border_width;
+    background: linear-gradient(to right, @c1 -40px,@c1 @sep,@c2 @sep,@c2 100%);
 }
 
+
 div.cell {
-    border: @cell_border_width solid transparent;
     .vbox();
     .corner-all();
     .border-box-sizing();
+    
     border-width: @cell_border_width;
     border-style: solid;
+    border-color: transparent;
+    transition: background 1s ease-out;
```

Comment:
Is there a reason for this super-slow transition? Seems to me like the change should be instant, or at least very-fast.